stable/signalfx-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: extensions/v1beta1
```

Comment:
`apps/v1beta2`

Reply:
Is it permissible to do a conditional here to support K8s 1.7 and older?  Or some other way to handle that?

Reply:
As the Charts only support the last three stable versions, we now only support 1.8 to 1.10, right?

Reply:
Ok, we'll just tell people the helm chart is only for newer versions, shouldn't be a problem.